test/polytopes.jl

```diff
@@ -43,6 +43,12 @@
     t = Triangle(P2(0.4,0.4), P2(0.6,0.4), P2(0.8,0.4))
     @test P2(0.2,0.4) ∉ t
     t = Triangle(P2(0,0), P2(1,0), P2(0,1))
+    @test t(T(0.0), T(0.0)) == P2(0,1)
+    @test t(T(0.0), T(1.0)) == P2(1,0)
+    @test t(T(1.0), T(0.0)) == P2(0,0)
+    @test t(T(0.5), T(0.5)) == P2(0.5,0.0)
```

Comment:
After adjusting the reference vertices, these tests will feel more intuitive with (u,v) --> (u,v)